enkit/outputs/commands.go

```diff
@@ -64,13 +67,6 @@ func NewRoot(base *client.BaseFlags) (*Root, error) {
 		BaseFlags: base,
 	}
 
-	homeDir, err := os.UserHomeDir()
-	if err != nil {
-		return nil, fmt.Errorf("failed to detect $HOME: %w", err)
-	}
-	defaultOutputsRoot := filepath.Join(homeDir, "outputs")
-
-	rc.PersistentFlags().StringVar(&rc.OutputsRoot, "outputs-root", defaultOutputsRoot, "Root dir of mounted outputs")
 	rc.PersistentFlags().StringVar(&rc.BuildBuddyApiKey, "api-key", "", "build buddy api key used to bypass oauth2")
 	rc.PersistentFlags().StringVar(&rc.BuildBuddyUrl, "buildbuddy-url", "", "build buddy url instance")
 	rc.PersistentFlags().StringVar(&rc.BuildbarnHost, "buildbarn-host", "", "host:port of BuildBarn instance")
```

Comment:
Are buildbarntunneltarget, tunnellistenport, and gatewayproxy below still used anywhere? If not, we should remove the flags.

Reply:
You're right that stuff isn't used anymore - I'll remove it.